xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-test/xwiki-platform-image-lightbox-test-pageobjects/src/main/java/org/xwiki/image/lightbox/test/po/Lightbox.java

```diff
@@ -0,0 +1,132 @@
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.image.lightbox.test.po;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.attributeToBe;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
+import org.xwiki.test.ui.po.ViewPage;
+
+public class Lightbox extends ViewPage
+{
+    public boolean waitUntilIsSlideDisplayed(int index)
+    {
+        try {
+            By slideSelector = By.cssSelector("#blueimp-gallery .slide-active");
+            getDriver().waitUntilCondition(attributeToBe(slideSelector, "data-index", String.valueOf(index)));
+            return getDriver().findElement(slideSelector).isDisplayed();
+        } catch (TimeoutException e) {
+            return false;
+        }
+    }
+
+    public WebElement getSlide()
+    {
+        By slideSelector = By.cssSelector("#blueimp-gallery .slide-active");
+        return getDriver().findElement(slideSelector);
+    }
+
+    public String getSlideIndex()
+    {
+        By slideSelector = By.cssSelector("#blueimp-gallery .slide-active");
+        return getDriver().findElement(slideSelector).getAttribute("data-index");
+    }
+
+    public String getCaptionContent()
+    {
+        By captionSelector = By.cssSelector("#blueimp-gallery .caption");
+        return getDriver().findElement(captionSelector).getText();
+    }
+
+    public String getTitleContent()
+    {
+        By titleSelector = By.cssSelector("#blueimp-gallery .title");
+        return getDriver().findElement(titleSelector).getText();
+    }
+
+    public String getPublisherContent()
+    {
+        By publisherSelector = By.cssSelector("#blueimp-gallery .publisher");
+        return getDriver().findElement(publisherSelector).getText();
+    }
+
+    public String getDateContent()
+    {
+        By dateSelector = By.cssSelector("#blueimp-gallery .date");
+        return getDriver().findElement(dateSelector).getText().replace("on", "");
+    }
+
+    public WebElement getDownloadElement()
+    {
+        By downloadSelector = By.cssSelector("#blueimp-gallery .download");
+        return getDriver().findElement(downloadSelector);
+    }
+
+    public boolean isMissingImageOpen()
+    {
+        try {
+            return getDriver().findElement(By.cssSelector("#blueimp-gallery .slide-error")).isDisplayed();
+        } catch (NoSuchElementException e) {
+            return false;
+        }
+    }
+
+    public void clickEscape()
+    {
+        By escapeSelector = By.cssSelector("#blueimp-gallery .escape");
+        getDriver().findElement(escapeSelector).click();
+    }
+
+    public void clickNext()
+    {
+        By nextSelector = By.cssSelector("#blueimp-gallery .next");
+        getDriver().findElement(nextSelector).click();
+    }
+
+    public void clickPrev()
```

Comment:
```suggestion
    public void previous()
```

And you should wait for the previous slide to load.